server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java

```diff
@@ -456,6 +456,25 @@ public ResultJson createNamespace(NamespaceJson json, UserData user) {
         return ResultJson.success("Created namespace " + namespace.getName());
     }
 
+    public ResultJson verifyToken(String namespaceName, String tokenValue) {
+        var token = users.useAccessToken(tokenValue);
+        if (token == null) {
+            throw new ErrorResultException("Invalid access token.");
+        }
+
+        var namespace = repositories.findNamespace(namespaceName);
+        if (namespace == null) {
+            throw new NotFoundException();
+        }
+
+        var user = token.getUser();
+        if (!users.hasPublishPermission(user, namespace)) {
+            throw new ErrorResultException("Insufficient access rights for publisher: " + namespaceName);
```

Comment:
You are using the term "publisher" here. I wonder if it's not supposed to be "namespace".